src/Constants.py

```diff
@@ -58,6 +58,7 @@ class PaymentStatus(Enum):
     PAID = 1
     DONE = 2
     INJECTED = 3
+    SKIPPED = 4
```

Comment:
SKIP is a term used in calculations. I find this confusing, a different name may be used. That is being said, What is the distinction between skipped and fail?

Reply:
* PAID/DONE status is for payment items which are successfully applied and are (or are expected to be) part of the blockchain and can be queried using a block explorer. When a delegator has a payout with a status PAID that means the baker is able to provide a proof on the blockchain that the reward payment was done successfully.
* FAILED status is for payment items which failed at any stage of the payment pipeline (run_operation/forging/signing/pre_apply). The reasons for that depend on the individual case and bakers in general need support to solve that (support queries take time and effort).
* SKIPPED status is for payment items which are avoided with intention. In most cases due to the fact that costs of the transaction are higher than the reward amount. For many bakers, this transaction is not failed because it is avoided with intention and it is not DONE because the payout was not really performed by the baker, thus the idea of this new status.